sensor/kubernetes/sensor/sensor.go

```diff
@@ -63,7 +65,8 @@ var (
 func CreateSensor(cfg *CreateOptions) (*sensor.Sensor, error) {
 	log.Info("Running sensor with Kubernetes re-sync disabled")
 
-	storeProvider := resources.InitializeStore()
+	hm := heritage.NewHeritageManager(pods.GetPodNamespace(), cfg.k8sClient, time.Now())
```

Comment:
Q: Shouldn't the pod start time be taken from the k8s object instead of `time.Now()`?

Reply:
It could, but we don't need that level of precision and getting it from the k8s informer would require more code to be written.

For all other containers than Sensor, we would have to get it from the k8s object, but sensor is the only example where we may take this shortcut.